exercises/practice/acronym/acronym-tests.8th

```diff
@@ -1,3 +1,4 @@
+9 !test-count
```

Comment:
```suggestion
9 plan-test-count
```
To avoid confusing beginners, I'd suggest a more descriptive word

Reply:
However, I don't know that "plan-" means in this context. The "!" was chosen in the light of various collections of [FORTH naming conventions](https://github.com/henryhudson/forth/blob/main/Style-Conventions.txt). But if "!" isn't good, how about `set-test-count` or even `17 tests`?

Reply:
Yes, good point. "plan" is a term from one of perl's unit testing frameworks: "I plan to execute 17 tests". It's not a good fit here. `17 tests` looks great.